resources/diff/rules/defaultRules.json

```diff
@@ -164,5 +164,24 @@
         "changedProperty": "core:version"
       }
     }
+  },
+  {
+    "name": "Rule to ignore changes in examples",
+    "conditions": {
+      "any": [
+        {
+          "fact": "diff",
+          "path": "$.type",
+          "operator": "contains",
+          "value": "apiContract:Example"
+        }
+      ]
+    },
+    "event": {
+      "type": "example-change",
```

Comment:
change type to "example-changed" - just to be consistent with types in other rules

Reply:
Makes sense, done..